components/services/UploadService.tsx

```diff
@@ -0,0 +1,23 @@
+import mediaService from "../../common/mediaService";
+
+
+class UploadFilesService {
+  upload(file, onUploadProgress) {
+    let formData = new FormData();
+
+    formData.append("file", file);
+
+    return mediaService.post("/api/v1/send_file/", formData, {
+      headers: {
+        "Content-Type": "multipart/form-data",
+      },
+      onUploadProgress,
+    });
+  }
+
+  getFile(file_id) {
+    return mediaService.get(`/api/v1/get_file/${file_id}`);
+  }
+}
+
+export default new UploadFilesService();
```

Comment:
not sure I understand the need for these services being separated

Reply:
is to have more flexibility when making a request to these endpoints, as the service will accept any object as payload and return a generic promise, which allows defining custom post actions and exception catchers